src/util/RouterUtil.js

```diff
@@ -208,6 +208,9 @@ function (Okta, OAuth2Util, Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
         router.settings.callGlobalSuccess(Enums.SUCCESS, successData);
       }
       return;
+    case 'ADMIN_CONSENT_REQUIRED':
+      router.navigate('signin/consent', {trigger: true});
```

Comment:
do you also need to check `if (router.settings.get('features.consent'))` ?

Reply:
No, I explicitly want it decoupled from APIAM Consent